From khali@linux-fr.org Sat Nov 26 19:24:08 2005
Date: Sat, 26 Nov 2005 20:24:59 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Subject: [PATCH 07/25] i2c: Drop i2c_driver.flags, 1 of 3
Message-Id: <20051126202459.080d3091.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-flags-01-df-dummy.patch

The I2C_DF_DUMMY flag is gone since 2.5.70, it's about time to
drop all ifdef'd out references thereto.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/media/video/tvmixer.c |    4 ----
 include/linux/i2c.h           |    5 -----
 2 files changed, 9 deletions(-)

--- gregkh-2.6.orig/drivers/media/video/tvmixer.c
+++ gregkh-2.6/drivers/media/video/tvmixer.c
@@ -232,12 +232,8 @@ static struct i2c_driver driver = {
 #endif
 	.name            = "tv card mixer driver",
 	.id              = I2C_DRIVERID_TVMIXER,
-#ifdef I2C_DF_DUMMY
-	.flags           = I2C_DF_DUMMY,
-#else
 	.flags           = I2C_DF_NOTIFY,
 	.detach_adapter  = tvmixer_adapters,
-#endif
 	.attach_adapter  = tvmixer_adapters,
 	.detach_client   = tvmixer_clients,
 };
--- gregkh-2.6.orig/include/linux/i2c.h
+++ gregkh-2.6/include/linux/i2c.h
@@ -252,11 +252,6 @@ static inline void i2c_set_adapdata (str
 
 /*flags for the driver struct: */
 #define I2C_DF_NOTIFY	0x01		/* notify on bus (de/a)ttaches 	*/
-#if 0
-/* this flag is gone -- there is a (optional) driver->detach_adapter
- * callback now which can be used instead */
-# define I2C_DF_DUMMY	0x02
-#endif
 
 /*flags for the client struct: */
 #define I2C_CLIENT_ALLOW_USE		0x01	/* Client allows access */
